app/src/main/java/nl/tudelft/b_b_w/controller/ConversionController.java

```diff
@@ -3,8 +3,9 @@
 import java.security.MessageDigest;
 
 /**
- * Class to convert values into a hashed value
+ * Class to convert values into a hashed value. Use BlockData.calculateHash instead.
```

Comment:
Be careful with putting everything in a BlockData class (Single responsibility principle).